### ISSUE TITLE
Cannot convert SSD resnet101 from tensorflow model zoo to ONNX

### ISSUE THREAD

**Issue by alexm-nm** [onnx/tensorflow-onnx#894]
Attempting to convert the SSD resnet101 (http://download.tensorflow.org/models/object_detection/ssd_resnet101_v1_fpn_shared_box_predictor_oid_512x512_sync_2019_01_20.tar.gz) model from the tensorflow model zoo to ONNX, generates the following exception:

```
Traceback (most recent call last):
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/tfonnx.py", line 266, in tensorflow_onnx_mapping
    func(g, node, **kwargs)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/onnx_opset/tensor.py", line 1506, in version_11
    cls.version_10(ctx, node, **kwargs)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/onnx_opset/tensor.py", line 1495, in version_10
    outputs=[node.output[0]], dtypes=dtypes[0], shapes=shapes[0])
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/graph.py", line 539, in make_node
    utils.make_sure(n is None, "name %s already exists in node: \n%s", name, n)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/utils.py", line 194, in make_sure
    raise ValueError("make_sure failure: " + error_msg % args)
ValueError: make_sure failure: name cast_A already exists in node: 
input: "Pad__1360:0"
output: "Postprocessor/BatchMultiClassNonMaxSuppression/map/while/MultiClassNonMaxSuppression/non_max_suppression_padded_99/NonMaxSuppressionV4:0"
name: "cast_A"
op_type: "Cast"
attribute {
  name: "to"
  i: 6
  type: INT
}
domain: ""


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/convert.py", line 169, in <module>
    main()
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/convert.py", line 153, in main
    inputs_as_nchw=args.inputs_as_nchw)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/tfonnx.py", line 482, in process_tf_graph
    mapped_op, unmapped_op, exceptions = tensorflow_onnx_mapping(g, ops_mapping)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/tfonnx.py", line 257, in tensorflow_onnx_mapping
    m_ops, unm_ops, body_exceptions = tensorflow_onnx_mapping(b_g, ops_mapping)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/tfonnx.py", line 269, in tensorflow_onnx_mapping
    logger.error("Failed to convert node %s\n%s", node.name, node.summary, exc_info=1)
  File "/home/alexm/tf2onnx/tensorflow-onnx/tf2onnx/graph.py", line 182, in summary
    node = g.get_node_by_output(name)
AttributeError: 'NoneType' object has no attribute 'get_node_by_output'

```
Command used is:

`python -m tf2onnx.convert     --input ~/onnx/ssd_resnet101/ssd_resnet101_v1_fpn_shared_box_predictor_oid_512x512_sync_2019_01_20/frozen_inference_graph.pb     --inputs 'image_tensor:0'     --outputs 'detection_boxes:0,detection_scores:0,num_detections:0,detection_classes:0'     --opset 11     --fold_const     --output ~/onnx/ssd_resnet101/ssd_resnet101_v1_fpn_shared_box_predictor_oid_512x512_sync_2019_01_20/frozen_inference_graph_ssd_resnet101_v1_fpn_shared_box_predictor_oid_512x512.onnx
`

**Urgency**
Very important

**System information**
- Ubuntu 16.04
- tensorflow version 1.15
- onnx version 1.6.0
- tf2onnx version 1.6.0/82f805
- python version 3.6
- opset used is 11

**Comment by jignparm** [comment]
Fixed in #895 

###Log
```
$ python -m tf2onnx.convert --input frozen_inference_graph.pb --inputs image_tensor:0 --outputs detection_boxes:0,detection_classes:0,detection_scores:0,num_detections:0 --output model.onnx --verbose --fold_const --opset 11

2020-04-24 07:25:24,824 - INFO - tensorflow: Froze 0 variables.
INFO:tensorflow:Converted 0 variables to const ops.
2020-04-24 07:25:25,372 - INFO - tensorflow: Converted 0 variables to const ops.
2020-04-24 07:25:33,917 - INFO - tf2onnx: inputs: ['image_tensor:0']
2020-04-24 07:25:33,917 - INFO - tf2onnx: outputs: ['detection_boxes:0', 'detection_classes:0', 'detection_scores:0', 'num_detections:0']
2020-04-24 07:25:36,120 - INFO - tf2onnx.tfonnx: Using tensorflow=2.1.0, onnx=1.6.0, tf2onnx=1.6.0/82f805
2020-04-24 07:25:36,120 - INFO - tf2onnx.tfonnx: Using opset <onnx, 11>
2020-04-24 07:26:12,184 - VERBOSE - tf2onnx.tfonnx: Mapping TF node to ONNX node(s)
...
...
2020-04-24 07:48:21,672 - INFO - tf2onnx.optimizer: After optimization: Add -1 (1250->1249), BatchNormalization -22 (24->2), Cast -1817 (5489->3672), Const -3454 (10404->6950), Gather +15 (1216->1231), Identity -61 (62->1), Less -2 (611->609), Mul -2 (624->622), Reshape +1 (623->624), Shape -2 (624->622), Slice -1 (1227->1226), Squeeze -20 (647->627), Transpose -373 (395->22), Unsqueeze -676 (2494->1818)
2020-04-24 07:48:27,220 - INFO - tf2onnx:
2020-04-24 07:48:27,220 - INFO - tf2onnx: Successfully converted TensorFlow model frozen_inference_graph.pb to ONNX
2020-04-24 07:48:28,016 - INFO - tf2onnx: ONNX model is saved at model.onnx
```

**Comment by alexm-nm** [comment]
Thanks @jignparm !
